content/news/buzz/december-2023-status-update/contents.lr

```diff
@@ -0,0 +1,60 @@
+title: December 2023 Status Update
+---
+author: Russell Keith-Magee
+---
+body:
+
+December has been a short month due to end-of-year holidays, but BeeWare has still made some important progress.
+
+What we've done
+----------------
+
+* `PEP730 <https://peps.python.org/pep-0730/>`__, formalizing the introduction of iOS as a Tier 3 platform in CPython, has been `ratified by the CPython Steering Council <https://github.com/python/steering-council/issues/218#issuecomment-1839333665>`__. 
+
+* The `first patch to CPython that explicitly mentions iOS <https://github.com/python/cpython/pull/112738>`__ has been merged. This is a relatively minor change that is required for building cross-platform binary modules; we'll start submitting more substantial patches in the new year.
+
+* `PEP738 <https://peps.python.org/pep-0738/>`__, formalizing the introduction of Android as a Tier 3 platform in CPython, has been drafted, and is `currently under discussion <https://discuss.python.org/t/pep-738-adding-android-as-a-supported-platform/40975>`__.
```

Comment:
This returns a 404 currently... not clear how long that should be the case though...

The same URL is used a bit further down as well.

Reply:
Looks like whatever problem you were having, it was transient; the URL is working for me now.

Reply:
The PEP still 404s for me.

https://peps.python.org/pep-0738/

Reply:
Oh bother... Github's UI FTW. I thought you were referring to the discussion link. 

And yes, the PEP can't possibly resolve (at least, not yet) because the PEP hasn't been merged. When it *is* merged, that's the link that will exist.

Unfortunately, I've now published the post... given that the link *will* be valid in a week or two, my inclination is to leave it.

What's weird is that I *thought* I had a link checker enabled on Lektor...